src/parser/transforms.js

```diff
@@ -267,12 +267,12 @@ module.exports.preprocessGlimmerTemplates = function preprocessGlimmerTemplates(
         let start = n.range[0];
         let codeSlice = code.slice(...n.range);
         for (const part of n.tag.split('.')) {
-          const regex = new RegExp(`\\b${part}\\b`);
+          let pattern = `\\b${part}\\b`;
+          if (part.startsWith(':')) {
```

Comment:
What's the significance of the leading \\b vs not?

Reply:
` \b` is only an assertion that we are at a word boundary. so either before or after there is a word character.
But with named block, we are not at a word boundary since there are no word characters at the beginning. `:` is not a word character. but with `:` we are sure that we match the correct part